src/app/lorax/directives/modal-email.js

```diff
@@ -92,12 +92,40 @@ define(['angular', 'jquery', 'jquery-customselect'], function (angular, $) {
          (this._$scope.modalEmail.terms && this._$scope.modalEmail.email) ? true : false;
     };
 
-    ModalEmailController.prototype.onSubmitFail = function () {
-        this._$scope.modalEmail.showFailedBtn = true;
-    };
-
-    ModalEmailController.prototype.onSubmitSuccess = function () {
-        this._$scope.modalEmail.showSuccessBtn = true;
+    ModalEmailController.prototype.onFormSubmit = function () {
+        var url = 'https://basket.allizom.org/news/subscribe/';
+        var newsletters = 'shape-web';
+        var source_url = 'https://shapeoftheweb.mozilla.org';
+        source_url = encodeURIComponent(source_url);
+        // store a reference to the controller
+        var controller = this;
+        var email = controller._$scope.modalEmail.email;
+        var country = controller._$scope.modalEmail.country;
+        var lang = 'en'
+        var fmt = 'H';
+        var privacy = 'on';
+        var params = 'newsletters=' + newsletters + '&source_url=' + source_url + '&email=' + email +
+            '&country=' + country + '&lang=' + lang + '&fmt=' + fmt + '&privacy=' + privacy;
+        controller._$scope.modalEmail.showSubmitBtn = false;
+        var r = new XMLHttpRequest();
```

Comment:
Just because it is in here already and used in other places, perhaps you can redo this to use jQuery?